modules/core/src/test/java/org/apache/ignite/internal/processors/cache/WalModeChangeAdvancedSelfTest.java

```diff
@@ -56,11 +56,11 @@ public WalModeChangeAdvancedSelfTest() {
 
     /** {@inheritDoc} */
     @Override protected void afterTest() throws Exception {
-        super.afterTest();
-
         stopAllGrids();
 
         cleanPersistenceDir();
+
+        super.afterTest();
```

Comment:
The change doesn't look sane to me.
Call to super does some additional validation that makes no sence on stopped grid.

Reply:
Oh, it is a buggy logic.
Returned as it has been.